src/core/lib/gpr/alloc.h

```diff
@@ -25,4 +25,12 @@
 #define GPR_ROUND_UP_TO_ALIGNMENT_SIZE(x) \
   (((x) + GPR_MAX_ALIGNMENT - 1u) & ~(GPR_MAX_ALIGNMENT - 1u))
 
+#define GPR_ROUND_UP_TO_CACHELINE_SIZE(x) \
```

Comment:
Maybe define `GPR_ROUND_UP_TO_ALIGNMENT_SIZE()` and `GPR_ROUND_UP_TO_CACHELINE_SIZE()` as invocations of `GPR_ROUND_UP_TO_SPECIFIED_SIZE()`?

Reply:
Done.